### PR TITLE
Switched project name from directory name to package.json name attribute

### PR DESCRIPTION
**Pull request by lannonbr**

### CODE REVIEW

**Comment by Hypercubed** [comment]
can you please do the following:

1) Add a fall back to use the directory name if package.json is not found or missing a name.
2) Add a /test/fixtures/package.json file.

Either one of these would pass the tests... but we should do both.

**Comment by Hypercubed** [comment]
LGTM. cc/  @stefanjudis

**Comment by stefanjudis** [comment]
Why not use simple `require` for reading the `package.json`?

Also not exactly sure, but does this line

```
projectName = JSON.parse( fs.readFileSync( creditPath + '/package.json' ) ).name;
```

work properly on windows? Having `/` 's in paths usually causes troubles. :)

But yeah overall looks good. :)

👍 

@Hypercubed @lannonbr

File: cli.js [hunk 1]
```diff
@@ -53,6 +53,11 @@ credits( creditPath )
  * @param  {Array} credits credits
  */
 function printCredits( credits ) {
-  let projectName = JSON.parse( fs.readFileSync( creditPath + '/package.json' ) ).name;
+  let projectName = '';
+  try {
+    projectName = JSON.parse( fs.readFileSync( creditPath + '/package.json' ) ).name;
```
Comment (Hypercubed) [inline cli.js]: replace with:

```
projectName = require( path.join( creditPath, '/package.json' ) ).name;
```

File: test/fixtures/package.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "name": "fixtures",
```
Comment (Hypercubed) [inline test/fixtures/package.json]: Please change to 

```
  "name": "credits-cli-test",
```

and add check to `test/cli.spec.js`.

**Comment by Hypercubed** [comment]
@stefanjudis Seams to work fine on windows... but to be safe I made a few comments.

@lannonbr If you can please address the comments below.  Also in `test/cli.spec.js` please add the line:

```
t.deepEqual( result[ 0 ] ), 'Credits for credits-cli-test' )
```

To match the change in the package.json.

Thank you for your contribution.

**Review by Hypercubed — CHANGES_REQUESTED** [review]

File: cli.js [hunk 3]
```diff
@@ -53,6 +53,11 @@ credits( creditPath )
  * @param  {Array} credits credits
  */
 function printCredits( credits ) {
-  let projectName = creditPath.split( path.sep ).pop();
+  let projectName = '';
+  try {
+    projectName = JSON.parse( fs.readFileSync( creditPath + '/package.json' ) ).name;
```
Comment (Hypercubed) [inline cli.js]: `projectName = require( path.join( creditPath, '/package.json' ) ).name;`

**Review by Hypercubed — CHANGES_REQUESTED** [review]

File: cli.js (hunk 3 above)
Comment (stefanjudis) [inline cli.js]: Could we just use `require` here?

**Comment by stefanjudis** [comment]
LGTM. 🚢

**Review by Hypercubed — APPROVED** [review]

**Pull request merged**